Worktrolly/worktrolly
# Dev/header

### Functionality:
Makes Navbar more funtional.

### Solution:
Grouped the create task and start sprint buttons and grouped the board buttons and added button to create team and add members. Added focus nav to the board component.

### Risk level:

### How to test:
1. Use ng serve and see the difference in the Navbar.

Picture of navbar:

![image](https://user-images.githubusercontent.com/58979519/107970437-1cfe1f00-6fd7-11eb-9d62-abbe6250e4dd.png)
